extraction/extractor.py

```diff
@@ -84,8 +85,10 @@ def __extract_data_from_html_body(self, node):
     def __is_content(self, node):
         """This function will extract valid content and return it"""
 
-        # This function will be updated in next iteration
-        _ = node
         _ = self.soup
+        # checks if content is text
+        is_content, text_content = text_extractor.is_text(node)
+        if is_content:
+            return True, text_content
 
```

Comment:
What is the value of text_content when is_content is False? If it is always {}, then do we need this condition here?